merge-guide.md

```diff
@@ -0,0 +1,47 @@
+# Procedure for adding new features to webrtc-pc
+
+The WebRTC-PC spec is now a W3C Rec..
+
+This means that we should revise it no more often than every 6 months or so, and only
+incorporate new features that are appropriate for "recycle at REC".
+
+This means that we have, at any given moment, 3 different documents to consider:
+
+* WebRTC-PC (REC)
+* WebRTC-PC (Editors' draft)
+* WebRTC-Extensions
+
+## Incubation
+
+In order to have a new feature considered for inclusion, write it up as an extension spec
+or as a pull request against webrtc-extensions.
+
+## Landing in webrtc-extensions
+
+We will land extensions in webrtc-extensions on the following criteria:
+
+* There is rough consensus in the WG that this API makes sense to consider
+* There is commitment from at least one browser vendor that they intend to implement
```

Comment:
I think this should also say what goes into webrtc-extensions vs an extension spec - my sense is anything that can be defined cleanly as an extension spec should be defined that way (e.g. doesn't require monkey patching algorithms).

Reply:
I am thinking that "large features" (webrtc-encoded-transform) should be their own spec. "Small features" (priority would have been one if -extensions existed at the time, header control is one) should go into the extensions spec.

Monkey-patching will be equally hard in extensions and in standalone specs.